mlir/include/mlir/Dialect/SparseTensor/IR/Enums.h

```diff
@@ -229,54 +263,51 @@ constexpr const char *toMLIRString(LevelType lt) {
     return "loose_compressed(nonordered)";
   case LevelType::LooseCompressedNuNo:
     return "loose_compressed(nonunique, nonordered)";
-  case LevelType::TwoOutOfFour:
-    return "block2_4";
+  default:
```

Comment:
can't this be a simple case on NOutOfM level type?
the other bits should not impact the enum, right?
perhaps by ending out the relevant part

Reply:
It would not catch n:m with n and m set. I could do `auto lt = static_cast<LevelType>(static_cast<uint64_t>(lvlType) & 0xffffffff);` and use `case LevelType::NOutOfM`. But since there's a function isNOutOfM, I chose that for readability. But I don't have a strong preference, so if you prefer having a simple case, I can do that as well.

Reply:
Yeah, I think we should use the mask + simple case for this